awstest/modules/task/module.ftl

```diff
@@ -0,0 +1,127 @@
+[#ftl]
+
+[@addModule
+    name="task"
+    description="Testing module for the aws ecs task component"
+    provider=AWSTEST_PROVIDER
+    properties=[]
+/]
+
```

Comment:
This doesn't appear to be testing tasks components and is actually testing the service component based on the solution module.

Reply:
Damn C&P error - sorry